### ISSUE TITLE
Tilted sky with Continuum shaders

### ISSUE THREAD

**Issue by HariboTer** [Abastro/StellarSky#106]
I tried out whether this mod would work with OptiFine and the Continuum shaderpack (version 2.02) which seemed to be successful after setting Render_Other_Modded_Skybox to true for all dimensions, but unfortunately the skybox was rotated by 45 degree which resulted in a quarter of it always being greyish or dark and sun and moon taking rather interesting routes while travelling over the continent inbetween funky looking dawns and dusks.
I'd be great if you could take a look into this since I really like the idea of having a gorgeous sky with gorgeous shaders^^

<details> 
  <summary>Some screenshots to demonstrate the problem</summary>

[2019-01-14_00 54 50](https://user-images.githubusercontent.com/3805624/51092659-1a96be80-179a-11e9-9023-a6f7f921de48.png)

[2019-01-14_00 55 27](https://user-images.githubusercontent.com/3805624/51092661-1a96be80-179a-11e9-8e63-1c4dbc9645d1.png)

[2019-01-14_00 55 35](https://user-images.githubusercontent.com/3805624/51092662-1b2f5500-179a-11e9-8791-de5ca08f2abf.png)

[2019-01-14_00 56 21](https://user-images.githubusercontent.com/3805624/51092657-1a96be80-179a-11e9-884b-6e835efc5db4.png)
</details>

**Comment by HariboTer** [comment]
The same problem happens with Sildur's Shaders, too. I assume it applies to others as well, I just haven't tested more (and am quite happy with Sildur's so far)^^

**Comment by stonemandave** [comment]
I agree it would be awesome if we could find away to keep stars and moon and also be able to have everything from the shader except they're sun and moon and stars. I'm not a modder but i wish i can help
